.github/workflows/release.yaml

```diff
@@ -238,38 +278,43 @@ jobs:
           - 'v1.23.6'
           - 'v1.24.2'
           - 'v1.25.3'
+        test: ${{ fromJSON(needs.setup-e2e-tests.outputs.test_names) }}
```

Comment:
This N*M (=75 currently) exceeds the limit of GH subnet, so they could not run at the same time. We need some concurrency control method.